packages/webapp/src/routes/__tests__/developer/check-answers.spec.js

```diff
@@ -1,12 +1,156 @@
-import { submitGetRequest } from '../helpers/server.js'
+import setDeveloperApplicationSession from '../../../__mocks__/developer-application-session.js'
+import checkAnswers from '../../../routes/developer/check-answers.js'
 import constants from '../../../utils/constants.js'
 const url = constants.routes.DEVELOPER_CHECK_ANSWERS
+jest.mock('../../../utils/http.js')
 
 describe(url, () => {
   describe('GET', () => {
-    it(`should render the ${url.substring(1)} view`, async () => {
-      const response = await submitGetRequest({ url })
-      expect(response.statusCode).toBe(200)
+    it(`should render the ${url.substring(1)} view`, done => {
+      jest.isolateModules(async () => {
+        try {
+          const session = setDeveloperApplicationSession()
```

Comment:
btw submitGetRequest now automatically populates with session data so that routes needing session data don't fail on server.inject, you can add additional fields to the session object for the developer journey if necessary
see webapp/src/__mocks__/on-pre-handler.js